core/src/apps/cardano/sign_tx.py

```diff
@@ -642,7 +643,9 @@ async def _process_withdrawals(
         )
         previous_reward_address = reward_address
 
-        await confirm_withdrawal(ctx, withdrawal)
+        await confirm_withdrawal(
+            ctx, withdrawal, encode_human_readable_address(reward_address)
```

Comment:
I would suggest the following. Rename `reward_address` to `reward_address_bytes` and pass that into`confirm_withdrawal` where the parameter would also be renamed to `reward_address_bytes`. And `encode_human_readable_address` would be called in `confirm_withdrawal` instead.

Maybe the rename is not necessary since it's clear from the types what it contains.

Reply:
Changed in https://github.com/vacuumlabs/trezor-firmware/pull/71/commits/9119a7472389bf7dd3640d9653bf04b388112de3